docs/user_guide/config_options.md

```diff
@@ -29,107 +29,144 @@ The different running modes of `benchcab` are solely dependent on the options us
     This running mode is quite open to customisations:
 
     * Specify the number of CABLE's branches you need
-    * Use [`patch`](#`patch`) on branches as required
-    * Specify the [science configurations](#`science_configurations`) you want to run. [`patch`](#`patch`) will be applied on top of the science configurations listed.
+    * Use [`patch`](#+realisation.patch) on branches as required
+    * Specify the [science configurations](#`science_configurations`) you want to run. [`patch`](#+realsation.patch) will be applied on top of the science configurations listed.
 
 
-## Technical options
+## project
 
-### `project`
+: **Default:** _required option, no default_. :octicons-dash-24: NCI project ID to charge the simulations to.
 
-: NCI project ID to charge the simulations to.
+``` yaml
 
-### `modules`
+project: nf33
 
-: NCI modules to use for compiling CABLE
+```
 
-### `fluxsite`
-: Contains settings specific to fluxsite tests. 
-: This key is **optional**. Default settings for the fluxsite tests will be used if it is not present
+## modules
 
-#### `pbs`
+: **Default:** _required option, no default_. :octicons-dash-24: NCI modules to use for compiling CABLE
 
-: Contains settings specific to the PBS scheduler at NCI for the PBS script running the CABLE simulations at FLUXNET sites and the bitwise comparison for these simulations. 
-: This key is **optional**. Default values for the PBS settings will apply if it is not specified.
+``` yaml
 
-##### `ncpus`
+modules: [
+  intel-compiler/2021.1.1,
+  netcdf/4.7.4,
+  openmpi/4.1.0
+]
 
-: The number of CPU cores to allocate for the PBS job, i.e. the `-l ncpus=<4>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]). 
-: This key is **optional** and can be omitted from the config file. By default `ncpus` is set to `18`.
+```
 
-##### `mem`
+## experiment
 
-: The total memory limit for the PBS job, i.e. the `-l mem=<10GB>` PBS flag (see [PBS Directives Explained][nci-pbs-directives]).
-: This key is **optional** and can be omitted from the config file. By default `mem` is set to `30GB`.
+: **Default:** _required option, no default_. :octicons-dash-24: Type of experiment to run. Experiments are defined in the **benchcab-evaluation** workspace on [modelevaluation.org][meorg]. This key specifies the met forcing files used in the test suite. To choose from:
 
-##### `walltime`
+    - [`forty-two-site-test`][forty-two-me]: to run simulations using 42 FLUXNET sites
```

Comment:
Is there anything stopping us from putting this in a table?

i.e.

| Key | Simulation site(s) |
|-|-|
|`AU-Tum`| Tumbarumba (AU) |